annotator/static/views/rect.js

```diff
@@ -323,6 +364,10 @@ class Rect {
         return this.boundsBeforeDrag != null;
     }
 
+    isBeingDraggedViaKeydown() {
```

Comment:
I'm putting it here because it's not clear that the absMouseBeforeDrag is being used for two purposes. Writing such a function clarifies this dual functionality, sort of